include/zenoh-pico/config.h

```diff
@@ -21,7 +21,7 @@
 #define Z_BATCH_MULTICAST_SIZE 2048
 #define Z_CONFIG_SOCKET_TIMEOUT 100
 
-#define Z_FEATURE_UNSTABLE_API 0
+#define Z_FEATURE_UNSTABLE_API 1
```

Comment:
I'd revert this value as PIO doesn't seem to autogenerate this file properly for now. Otherwise, LGTM